src/style.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+}
+
+body p {
+  margin: 0;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 14px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-image {
+  margin: 15px 6px 41px;
+}
+
+.product-image__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product-description {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  text-align: left;
+  margin-bottom: 6px;
+}
+
+.product-code {
+  margin-bottom: 15px;
+  width: 100%;
+}
+
+.product-code__text {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  text-align: left;
+  line-height: 13px;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.review__stars {
+  display: flex;
+}
+
+.review__text {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
```

Comment:
You use `font-size: 11px;` three times in your code. Don't repeat yourself. Let's move it to `body`